js/src/views/Dapps/dappsStore.js

```diff
@@ -210,39 +202,43 @@ export default class DappsStore {
       });
   }
 
-  _fetchLocal () {
-    return fetch(`${this._getHost()}/api/apps`)
-      .then((response) => {
-        return response.ok
-          ? response.json()
-          : [];
-      })
-      .then((localApps) => {
-        return localApps
-          .filter((app) => app && app.id && !['ui'].includes(app.id))
-          .map((app) => {
-            app.type = 'local';
-            return app;
-          });
-      })
-      .catch((error) => {
-        console.warn('DappsStore:fetchLocal', error);
-      });
-  }
-
   _readHiddenApps () {
-    const stored = localStorage.getItem('hiddenApps');
+    const stored = localStorage.getItem(LS_KEY_HIDDEN);
 
     if (stored) {
       try {
-        this.hidden = JSON.parse(stored);
+        this.hiddenApps = JSON.parse(stored);
       } catch (error) {
         console.warn('DappsStore:readHiddenApps', error);
       }
     }
   }
 
+  _readExternalApps () {
+    const stored = localStorage.getItem(LS_KEY_EXTERNAL);
+
+    if (stored) {
+      try {
+        this.externalApps = JSON.parse(stored);
+      } catch (error) {
+        console.warn('DappsStore:readExternalApps', error);
+      }
+    }
+  }
+
+  _writeExternalApps () {
+    try {
+      localStorage.setItem(LS_KEY_EXTERNAL, JSON.stringify(this.externalApps));
```

Comment:
We actually have an issue logged to rework the localStorage - I prefer to do it properly everywhere and set the standard to doing it piecemeal and forgetting about the rest. (So basically when creating the store didn't re-invent the code that was there.)